packages/deployment/ag-setup-cosmos

```diff
@@ -1,13 +1,32 @@
 #! /usr/bin/env node
+/* global require, module, setInterval */
 
+const fs = require('fs');
+const path = require('path');
+const temp = require('temp');
+const process = require('process');
+const { exec, spawn } = require('child_process');
+const inquirer = require('inquirer');
+const fetch = require('node-fetch');
```

Comment:
All of these should be after `esmRequire('@agoric/install-ses')`.

Reply:
speaking of `esmRequire`... vs-code's "go to definition" feature works with it if we assign it to `require`.

b2c02e08c301